lib/old_roman.js

```diff
@@ -0,0 +1,27 @@
+const key = {
+  "M": 1000,
+  "D": 500,
+  "C": 100,
+  "L": 50,
+  "X": 10,
+  "V": 5,
+  "I": 1
+}
+
+function convertToOldRoman(num) {
+  let output = '';
+
+  for (var letter in key) {
+    if (key.hasOwnProperty(letter)) {
```

Comment:
We don't need this check. Since we are iterating through the keys we know the property will exist on the object.